source/conf.py

```diff
@@ -22,7 +22,7 @@
 from pathlib import Path
 
 project = 'Adventure'
-copyright = '2020-2025 KyoriPowered'
+copyright = '2020-2025 KyoriPowered. Not official Minecraft software. Not approved by or associated with Mojang or Microsoft.'
```

Comment:
Shouldn't it be Mojang Studios?
```suggestion
copyright = '2020-2025 KyoriPowered. Not official Minecraft software. Not approved by or associated with Mojang Studios or Microsoft.'
```

Reply:
This is what we have been advised to include by Mojang

Reply:
Alright, then ig not